ocs_ci/deployment/deployment.py

```diff
@@ -3752,7 +3752,7 @@ def deploy(self):
         config.switch_ctx(old_ctx)
         # Only on the active hub enable managedserviceaccount-preview
         acm_version = get_acm_version()
-
+        config.switch_acm_ctx()
```

Comment:
Shouldn't we then return back to old_ctx later on in this function as we do on line 3752?

Reply:
@petr-balogh , I have rearragend the ctx switches between acm and old ctx.

@prsurve , please let me know if this is ok

Reply:
yes LGTM